src/core/server/base.js

```diff
@@ -30,7 +30,20 @@ const version = path.join(config.get('basePath'), 'version.json');
 const isDeployed = config.get('isDeployed');
 const isDevelopment = config.get('isDevelopment');
 
-const errorString = 'Internal Server Error';
+const errorPageText = {
+  401: 'Unauthorized',
+  404: 'Not Found',
+  500: 'Internal Server Error',
```

Comment:
Yep, that's a known issue and will be fixed when we move to using a proper page.